torch/_dynamo/codegen.py

```diff
@@ -256,11 +256,18 @@ def create_load_attr(self, name):
     def create_load_attrs(self, names):
         return [self.create_load_attr(name) for name in names.split(".")]
 
-    def load_function_name(self, fn_name, num_on_stack=0):
+    def load_function_name(self, fn_name, push_null, num_on_stack=0):
```

Comment:
No you ever want push_null=False here?

Reply:
There might already be a null on the stack (e.g. the NULL popping method in symbolic_convert) or maybe we want to push a function without intending to call it in the future (don't think this is currently done anywhere).

Reply:
ok!